20 ricerca

scrivereleonardo opened the pull request:
Closes #20 #17

Scrivere committed:
Scritta prima bozza

Scritta la prima bozza di prova per la stampa di tutti gli Utenti:
Sarà necessario cambiare in futuro l'oggetto utilizzato per la bozza
poichè non è corretto ma solo un esempio.

Scrivere committed:
Aggiunte lievi modifiche

Scrivere committed:
Aggiunta prima bozza Ricerca/Visualizzazione

Ci sonono problemi nella ricerca tramite competenza al momento, per il
resto è tutto funzionate.

Scrivere committed:
Risolto problema su ricerca competenze

e' stato risolto il problema sulla ricerca delle competenze , e aggiunti
i filtri per i nomi delle aree, todo: fixare uno dei filtri delle aree

Scrivere committed:
Risolti problemi sulla ricerca delle competenze
la ricerc delle competenze delle aree dovrebbe funzionare correttamente.
Adesso le ricerche dovrebbero funzionare entrambe, ci sarà poi bisogno
di modificarle quando si otterranno tutti i dati dal server

Scrivere committed:
Aggiunto Primo Observable

E' stato aggiunto un observable per prendere un oggetto dalla Mock-list,
inoltre è ora possibile collegarsi alla pagina di modifica e di aggiunta
del singolo utente con un click sul suo nome.

TODO: Aggiungere un altro Observable per le competenze, e aggiungere
collegamento alla pagina di modifica e aggiunta delle Aree di
competenza, tramite click del loro nome sulla lista (Come è stato fatto
per gli utenti)

Scrivere committed:
Aggiunto Path alla pagina dell'Area di Comp.

E' stato aggiunto il percorso alla pagina della singola Area di competenza,
tramite la lista mostrata nelle Ricerca.
Aggiunti anche Fix Vari.
:

Scrivere committed:
Fix vari e Aggiunta di metodi

Aggiunti fix multipli, e aggiunti dei metodi che vanno a sostituire le
condizioni dentro gli IF, che erano troppo lunghi

Scrivere committed:
Ultime Modifiche

Wabri reviewed (changes requested):
- [x] il package-lock.json non è necessario perchè viene ricreato ogni volte che viene eseguita una build ng quindi puoi eliminarlo, quello importante è il package.json in cui ci sono tutte le configurazioni del progetto

scrivereleonardo commented on Angular/GestioneCompetenze/.browserslistrc:
```diff
@@ -0,0 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Intendi che posso proprio cancellare il file browserlistrc?

Wabri replied:
Si

Scrivere committed:
Effettuato Refactoring Codice

Adesso invece di avere lavorare con 2 oggetti, si lavora con tutti e 5
gli oggetti corretti (come specificato nella documentazione che si trova
su google drive) , inoltre sono stati modificati quasi tutti i metodi di
visualizzazione e ricerca. Infine c'è stata una lieve modifica anche
alla parte grafica
